cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -2417,5 +2417,6 @@ public static final class InternalRouter {
     public static final String BIND_ADDRESS = "app.program.internal.router.service.bind.address";
     public static final String BIND_PORT = "app.program.internal.router.service.bind.port";
     public static final String SSL_ENABLED = "app.program.internal.router.service.ssl.enabled";
+    public static final String USE_INTERNAL_ROUTER = "app.program.internal.router.use";
```

Comment:
Just notice it now. The InternalRouter are not app program specific, hence shouldn't be using that prefix. Also, we generally follow the inner class name as the prefix. I would suggest to change all these constants to `internal.router.` prefix.